index.html

```diff
@@ -1448,11 +1442,17 @@ <h2>
 
           PushSubscriptionJSON toJSON();
         };
+        
+        dictionary PushSubscriptionKeys
+        {
+          USVString p256dh;
+          USVString auth;
+        };
```

Comment:
Nit: consistency with indenting.

```suggestion
        dictionary PushSubscriptionKeys {
          USVString p256dh;
          USVString auth;
        };
```